examples/ChatQnA/deploy/gaudi.md

```diff
@@ -778,30 +812,30 @@ data: b'.'
 data: b'</s>'
 data: [DONE]
 ```
-### Guardrail Microservice
-If you had enabled Guardrail microservice, access via the below curl command
 
+##### Gaurdrail Microservice
+If you had enabled Gaurdrail microservice, access via the below curl command
 
-```

```

Comment:
These "codeblock" markers should not have been deleted.  Put the back before and after the bash command.

Reply:
Addressed. Thanks.